ui/lib/apps/SlowQuery/utils/useSlowQueryTableController.ts

```diff
@@ -165,6 +169,33 @@ export default function useSlowQueryTableController(
     getSlowQueryList()
   }, [queryOptions, orderOptions, queryTimeRange, refreshTimes, selectedFields])
 
+  const [downloading, setDownloading] = useState(false)
+
+  async function downloadCSV() {
+    try {
+      setDownloading(true)
+      const res = await client.getInstance().slowQueryDownloadTokenPost({
+        fields: '*',
+        db: queryOptions.schemas,
+        digest: queryOptions.digest,
+        text: queryOptions.searchText,
+        limit: queryOptions.limit,
```

Comment:
I think maybe we should cancel the limit when exporting, aka we should export all rows.